src/Command/LintTemplatesCommand.php

```diff
@@ -353,6 +364,11 @@ private function lintFamilies(array $families, array $familyCodesInIndustries):
                         case self::ATTRIBUTE_TYPE_PRICE_COLLECTION:
                             $this->assertValidBooleanProperty('decimals_allowed', $attribute, $index, $fileName, $violations);
                             break;
+                        case self::ATTRIBUTE_TYPE_TEXT:
+                            if (array_key_exists('validation_rule', $attribute)) {
```

Comment:
I'd like to suggest moving this condition also to the `assertValidAttributeValidationRule` method to encapsulate all the logic inside